OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -359,8 +351,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <material key="parchment"/>
                                     <p>A hole on <locus target="#54"/>, presumably from parchment
                                         making </p>
-                                    <p>A tear carefully amended on <q xml:lang="gez">ዘእኁድ</q>
-                                            (<locus target="#74r"/>)<locus target="#42"/>
+                                    <p>A tear carefully amended on <locus target="#42"/>
```

Comment:
I see that according to the guidelines everything concerning the material and its characteristics can go under `<material key="parchment"/>`(https://betamasaheft.eu/Guidelines/?q=condition&id=objectDescription).
However, I have so far discriminated between holes resulting from parchment making (which I encoded here) and tears/holes repaired (@Gnisci once corrected my "amended" into "repaired") by someone more or less carefully, which I encoded under Condition (in my view this seems to fit better that kind of feature).
We can discuss this here or in the next meeting and find a consistent solution.

Reply:
Holes are commonly already repaired during the parchment making process. See here: Winslow 2015, 109. If there are no hints for any later repair (like synthetic therads), I would group them here.